jadler-jdk/src/main/java/net/jadler/stubbing/server/jdk/JdkStubHttpServer.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2012 - 2015 Jadler contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package net.jadler.stubbing.server.jdk;
+
+import com.sun.net.httpserver.HttpServer;
```

Comment:
I can't find it anywhere (and I do not want to register to IBM just to download their JDK). But even if it were Oracle SDK only, it's an optional module so people do not have to use it.